cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -89,19 +89,20 @@ func IsNodeInfoSimilar(n1, n2 *schedulercache.NodeInfo) bool {
 		return false
 	}
 
+	ignoredLabels := map[string]bool{
+		kubeletapis.LabelHostname:             true,
+		kubeletapis.LabelZoneFailureDomain:    true,
+		kubeletapis.LabelZoneRegion:           true,
+		"beta.kubernetes.io/fluentd-ds-ready": true, // this is internal label used for determining if fluentd should be installed as deamon set. Used for migration 1.8 to 1.9.
+	}
+
 	labels := make(map[string][]string)
 	for _, node := range nodes {
 		for label, value := range node.Node().ObjectMeta.Labels {
-			if label == kubeletapis.LabelHostname {
-				continue
-			}
-			if label == kubeletapis.LabelZoneFailureDomain {
-				continue
-			}
-			if label == kubeletapis.LabelZoneRegion {
-				continue
+			ignore, _ := ignoredLabels[label]
```

Comment:
nit: if disregarding second returned value, you can just as well write `if !ignoredLabels[label] {`